ui/app/components/version-info/index.ts

```diff
@@ -0,0 +1,23 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+import ApiService from 'waypoint/services/api';
+import { Empty } from 'google-protobuf/google/protobuf/empty_pb';
+import { VersionInfo as info } from 'waypoint-pb';
+
+export default class VersionInfo extends Component {
+  @service api!: ApiService;
+  @tracked versionInfo: info.AsObject | undefined;
```

Comment:
I think if you assign this in the constructor you can actually not need the undefined, but maybe that would break the async behavior?

Reply:
I initially had that, but the type checker was yelling at me because `toObject` below returns `AsObject | undefined` (it also took me a while to figure out that was the issue)